articles/130_ros_time.md

```diff
@@ -39,6 +39,17 @@ Slower than real time simulation is necessary for complicated systems where accu
 Often the simulation is the limiting factor for the system and as such the simulator can be a time source for faster or slower playback.
 Additionally if the simulation is paused the system can also pause using the same mechanism.
 
+
+## Approach
+
+To provide a simplified time interface we will provide a ROS time and duration datatype.
+To query for the latest time a ROS Clock interface will be provided.
+The Clock will be populated by one or more TimeSource mechanisms.
```

Comment:
Not sure I understand the sentence. How can the clock be populated by more than one time source?

Reply:
Yeah, it's supposed to be one to many the other way. I inverted the logic and used the old language here.